package.json

```diff
@@ -56,7 +56,7 @@
     "@sapphire/async-queue": "^1.5.0",
     "@types/node-fetch": "^2.6.2",
     "@types/ws": "^8.5.3",
-    "discord-api-types": "^0.33.5",
+    "discord-api-types": "^0.37.11",
```

Comment:
Won't this cause a version clash with the older version of /builders?

Reply:
I guess we have to bring in those types manually 😓

Reply:
@vladfrangu I need a newer version to use `SortOrderType`

Reply:
https://github.com/discordjs/discord.js/pull/8651/commits/54267f584a4e38cb58421cf7d26785a5a621dfcd
oki 👍